librarymanagement/src/main/scala/sbt/librarymanagement/CrossVersionExtra.scala

```diff
@@ -13,33 +19,40 @@ abstract class CrossVersionFunctions {
   val TransitionSbtVersion = CrossVersionUtil.TransitionSbtVersion
 
   /** Cross-versions a module with the full version (typically the full Scala version). */
-  val full: CrossVersion = Full()
+  val full: CrossVersion = LFull()
 
   /**
    * Cross-versions a module with the result of prepending `prefix` and appending `suffix` to the full version.
    * (typically the full Scala version).  See also [[sbt.librarymanagement.Full]]
    */
-  def fullWith(prefix: String, suffix: String): CrossVersion = Full(prefix, suffix)
+  def fullWith(prefix: String, suffix: String): CrossVersion = LFull(prefix, suffix)
 
   /** Cross-versions a module with the binary version (typically the binary Scala version).  */
-  val binary: CrossVersion = Binary()
+  val binary: CrossVersion = LBinary()
 
   /** Disable crossversioning for the module. */
-  val disabled: CrossVersion = sbt.librarymanagement.Disabled()
-
-  /** Same as .disabled, for compatibility with 0.13 */
-  val Disabled: CrossVersion = sbt.librarymanagement.Disabled()
+  val disabled: CrossVersion = LDisabled()
+
+  /** Compatibility with 0.13 */
+  val Disabled: CrossVersion = disabled
+  val Binary: CrossVersion = binary
+  val Full: CrossVersion = full
+  val Patch: CrossVersion = patch
+  type Disabled = LDisabled
```

Comment:
There is no type `sbt.CrossVersion.Disabled` in sbt 0.13. There is only the singleton type `sbt.CrossVersion.Disabled.type`.